pkg/graveler/ref/kv_migrate_test.go

```diff
@@ -176,16 +204,48 @@ func verifyBranchResults(t *testing.T, ctx context.Context, kvMgr, dbMgr gravele
 		b := branches.Value()
 		bkv, err := kvMgr.GetBranch(ctx, repoID, b.BranchID)
 		require.NoError(t, err)
-
 		bdb, err := dbMgr.GetBranch(ctx, repoID, b.BranchID)
 		require.NoError(t, err)
 		require.Equal(t, bkv, bdb)
 		branchesNum += 1
+
+		// Skip staging check for main
```

Comment:
Why?

Reply:
We didn't add any data to the staging so we have nothing to validate (we rely on a given number of expected staging entries in the test)